cmd/serverutil/listen.go

```diff
@@ -0,0 +1,49 @@
+// Copyright 2020 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package serverutil
+
+import (
+	"context"
+	"net"
+
+	"github.com/golang/glog"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials"
+)
+
+// Listen binds to listenAddr and returns a gRPC connection to it.
+func Listen(ctx context.Context, listenAddr, certFile string) (net.Listener, *grpc.ClientConn, func()) {
+	// Listen and create empty grpc client connection.
```

Comment:
The "create empty grpc client connection" only happens further down - move this part of the comment down to where it happens to avoid confusion.